src/server.js

```diff
@@ -18,11 +21,16 @@ const fetchPage = async (data: { page: string }): string => {
 };
 
 export default () => async (req, res) => {
-    const pageType = req.params[0].split('/pages/')[0];
-    const data = require(`../.data/${pageType}`);
+    // just while we're not getting a state from play
+    const data = fakeState(req.params.page);
+
+    const { html: ignoreMe, ...config } = await fetch(
```

Comment:
I was just discussing `ignoreMe` with @SiAdcock whether we should prefix the name of variables with an `_` to denote that they're unused? Then we could update our lint rule `no-unused-vars` to ignore variables beggining with an `_`, as described at https://eslint.org/docs/2.0.0/rules/no-unused-vars#ignore-identifiers-that-match-specific-patterns